build.gradle

```diff
@@ -42,6 +42,8 @@ dependencies {
 	implementation 'com.google.guava:guava:29.0-jre'
 	implementation 'org.springframework:spring-jms:5.2.6.RELEASE'
 	implementation 'org.apache.qpid:qpid-jms-client:0.51.0'
+	implementation 'org.apache.commons:commons-collections4:4.0'
```

Comment:
Why do we need this additional dependency?

Reply:
Right, it's not needed. I though I'll use something from that but It don't. I'll remove that in next CR cycle